src/main/java/com/mySystem/Main.java

```diff
@@ -27,27 +32,14 @@ public static void main(String[] args) {
                 .get()
                 .build();//object chaining (class Builder has three methods url(),get(),build()
 
-//        try (Response response = client.newCall(request).execute()) {
-//            System.out.println(response.body().string());
-//        } catch (IOException e) {
-//            throw new RuntimeException(e);
-//        }
-
-        FileReader reader = null;
-        try {
-            reader = new FileReader("gmap_distance_matrix_response.json");
-            Gson gson = new Gson();
-            HashMap<String, Object> map = gson.fromJson(reader, HashMap.class);
-
-            ArrayList<Map<String, Object>> rows = (ArrayList<Map<String, Object>>) map.get("rows");
-            ArrayList<Map<String, Object>> elements = (ArrayList<Map<String, Object>>) rows.get(0).get("elements");
-            Map<String, Object> duration = (Map<String, Object>) elements.get(0).get("duration");
-            String durationText = (String) duration.get("text");
-
-            System.out.println("duration: " + durationText);
-          //  System.out.println(map);
-        } catch (FileNotFoundException e) {
+        try (Response response = client.newCall(request).execute()) {
```

Comment:
This request should only be made if we don't have the cached response (file with data) available.

Reply:
Done